product_docs/docs/pem/9/pem_rel_notes/930_rel_notes.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Version 9.3.0"
+---
+
+New features, enhancements, bug fixes, and other changes in PEM 9.3.0 include:
+
+| Type        | Description                                                                                      |
+| ----------- | -------------------------------------------------------------------------------------------------|
+| Enhancement | Added the functionality to delete all the existing alerts while using the copy alert feature.     |
+| Enhancement | Added PostgreSQL and EPAS 16 beta support as a backend database server and monitoring server.         |
+| Enhancement | Added support to customize the default email templates.                                          |
+| Enhancement | Added support to download the alert history report for agents and servers. |
+| Enhancement | Added options to reuse the existing agent SSL certificate and key file while configuring the PEM server. |
+| Bug Fix     | Added the functionality to add/view ignore mount points in the agent REST API.                  |
+| Bug Fix     | Added the `hostaddr` body parameter to the server REST API.                                     |
+| Bug Fix     | Fixed an issue by removing the newline character in the alert details from the payload of Webhook.                 |
+| Bug Fix     | Added support to allow the Webhook endpoint registration without providing the client SSL certificates and key files or CA certificates using `allow_insecure_webhooks` parameter in the `agent.cfg` file.|
+| Bug Fix     | Fixed an issue related to the backup sizes displaying on the PEM's Barman dashboard.           |
+| Bug Fix     | Fixed an issue with unformatted data while importing the audit logs.                           |
```

Comment:
should this just say "on the PEM Barman dashboard"? or "on Pem's Barman dashboard"?

Reply:
Done